vacationtime-solution/src/Code.js

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var COLUMN_NUMBER = {
+  EMAIL: 2,
+  NAME: 3,
+  MANAGER_EMAIL: 4,
+  START_DATE: 5,
+  END_DATE: 6,
+  APPROVAL: 8,
+  NOTIFIED: 9,
+};
+
+var REJECTION_EMAIL_MESSAGE = 'Your vacation time request was not approved.';
+var REJECTION_EMAIL_SUBJECT = 'ERR: Vacation Time Request NOT Approved';
+
+/**
+ * Add custom menu items when opening the sheet.
+ */
+function onOpen() {
+  var sheetUi = SpreadsheetApp.getUi();
+  sheetUi.createMenu('TimeOff')
+      .addItem('Manager Approval', 'managerApproval')
```

Comment:
Nit: Other solutions have generally settled on "Set up" for one-time set up tasks like this. Suggest aligning with that + update instructions.